### PR TITLE
Variants \ actions block

### PR DESCRIPTION
**Pull request by mpohorenyi**

### CODE REVIEW

File: src/modules/ProductDetailsPage/ProductDetailsPage.tsx [hunk 1]
```diff
@@ -31,26 +29,51 @@ function getDetails(
   if (color && capacity) {
     return productDetail.additional.find(
       (product) => product.color === color && product.capacity === capacity,
-    );
+    ) || null;
   }
```
Comment (OlhaKoziuk) [inline src/modules/ProductDetailsPage/ProductDetailsPage.tsx:33]: not clear to me why null? We enter in this block when we definatly sure that we have such params. Or we can possibly receive underfind?

File: src/modules/ProductDetailsPage/components/InfoAndPurchase/InfoAndPurchase.module.scss [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+    &__spec {
+      color: $color__dark-theme__white;
+
+      &__title {
+        color: $color__dark-theme__secondary;
+      }
+    }
+  }
+}
```
Comment (OlhaKoziuk) [inline src/modules/ProductDetailsPage/components/InfoAndPurchase/InfoAndPurchase.module.scss:79]: styles look good)))

**Review by OlhaKoziuk — APPROVED** [review]
LGTM)

**Review by dsfreedom — APPROVED** [review]
Everything looks good

File: src/modules/ProductDetailsPage/ProductDetailsPage.tsx (hunk 1 above)
Comment (mpohorenyi) [reply src/modules/ProductDetailsPage/ProductDetailsPage.tsx:33]: The find method can return undefined, so to better typify this whole thing, I add null

**Pull request merged**